2022/_data/jobs.yml

```diff
@@ -147,5 +147,69 @@
         "link": "https://www.linkedin.com/jobs/view/3068049755"
       }
     ]
+  },
+  {
+    "company": {
+      "name": "CloudBlue",
+      "logo-url": "/assets/img/sponsors/cloudblue.jpg",
+      "description": "CloudBlue, an Ingram Micro business, is the world’s largest cloud platform provider that orchestrates B2B ecosystems by enabling management of traditional and digital products and subscriptions, quickly onboarding, and integrating with partners, and scaling globally with a single platform.",
```

Comment:
There are a few typos, maybe we should double check with the company: `technology solution startup`, `Design and develop a for microservices`

Reply:
Typos are still there, @anyulled I understand we can't fix?

Reply:
Yes, fix them if you need to. No problem

Reply:
I only copied and pasted. I'll have a look later

Reply:
Let's merge to not delay, but not everything can be fixed by us. The last line seems to be missing a word and I cannot fathom which one :shrug: "a ??? from microservices".
![image](https://user-images.githubusercontent.com/5781153/174486713-d7c86ce1-6cbc-4291-af1e-f6bb466cd0a3.png)

It's in the interest of the company.

Reply:
Yes, that line could've been removed as well. But thanks, we can iterate on further modifications on this file.

Reply:
I'll do it in the next PR